docs/examples/use_cases/pytorch/efficientnet/readme.rst

```diff
@@ -133,19 +133,19 @@ To run training benchmarks with different data loaders and automatic augmentatio
   export RESULT_WORKSPACE=./
 
   # synthetic benchmark
-  python multiproc.py --nproc_per_node 8 ./main.py --amp --static-loss-scale 128 --batch-size 128 --epochs 1 --prof 1000 --no-checkpoints --training-only --data-backend synthetic --workspace $RESULT_WORKSPACE --raport-file bench_report_synthetic.json $PATH_TO_IMAGENET
+  python multiproc.py --nproc_per_node 8 ./main.py --amp --static-loss-scale 128 --batch-size 128 --epochs 1 --prof 1000 --no-checkpoints --training-only --data-backend synthetic --workspace $RESULT_WORKSPACE --report-file bench_report_synthetic.json $PATH_TO_IMAGENET
```

Comment:
`docs/examples/use_cases/pytorch/efficientnet/main.py` `"--raport-file"` needs to be adjusted as well.

Reply:
Oh, nice catch, thank you! Fixed in https://github.com/NVIDIA/DALI/pull/5194/commits/1abfa38d0c678771629ed8646a7e735824a683f1 . 

Btw @klecki, just to confirm, are we OK with breaking "backward compatibility" of this example?

Reply:
@stiepan ?

Reply:
I am not aware of some external automation that would rely on it. I think we should be good with the fix.

Reply:
Thank you!